packages/actions-test/src/event.ts

```diff
@@ -69,13 +73,31 @@ export class TestLog implements Log {
 }
 
 export class TestAlertEvent implements AlertEvent {
```

Comment:
I think this was added very recently and not used, so we can clear up TestAlertEvent. TestTransactionEvent should be used, if necessary.